analyzer/analyzer_test.go

```diff
@@ -347,13 +350,20 @@ func TestApplier_ApplyLayers(t *testing.T) {
 					Name:   "9.9",
 				},
 				Packages: []types.Package{
-					{Name: "libc6", Version: "2.24-11+deb9u4", SrcName: "glibc", SrcVersion: "2.24-11+deb9u4"},
-					{Name: "tzdata", Version: "2019a-0+deb9u1", SrcName: "tzdata", SrcVersion: "2019a-0+deb9u1"},
+					{
+						Name: "libc6", Version: "2.24-11+deb9u4", SrcName: "glibc", SrcVersion: "2.24-11+deb9u4",
+						LayerID: "sha256:dffd9992ca398466a663c87c92cfea2a2db0ae0cf33fcb99da60eec52addbfc5",
+					},
+					{
+						Name: "tzdata", Version: "2019a-0+deb9u1", SrcName: "tzdata", SrcVersion: "2019a-0+deb9u1",
+						LayerID: "sha256:932da51564135c98a49a34a193d6cd363d8fa4184d957fde16c9d8527b3f3b02",
+					},
 				},
 				Applications: []types.Application{
 					{
 						Type: "composer", FilePath: "php-app/composer.lock",
-						Libraries: []depTypes.Library{{Name: "guzzlehttp/guzzle", Version: "6.2.0"},
+						Libraries: []depTypes.Library{
+							{Name: "guzzlehttp/guzzle", Version: "6.2.0"},
```

Comment:
I'm sorry not to tell you. We also need to add `LayerID` to `Library`. A vulnerability is detected from `LIbrary` as well as `Package`. To show which layer this vulnerability comes from, we require `LayerID` in `Library`.

Reply:
Just to confirm the Library struct comes from this package? https://github.com/aquasecurity/go-dep-parser/blob/master/pkg/types/types.go 

If so we'll have to add it there. Sidenote: turns out that package as lots of un-named struct initializers so adding a new field will require fixing all of them 😞

Reply:
Ah, I forgot `Package` came from `go-dep-parser`. Hmm... `go-dep-parser` is a library not related to a container. So I'd like to avoid adding `LayerID` to `Package`. Do you have any idea? Our goal is to display a layer ID with a vulnerability.

Reply:
I went with creating a new struct and encapsulating the Library struct inside of it.